File: 2634-minimum-common-value/minimum-common-value.java
class Solution {
    public int getCommon(int[] nums1, int[] nums2) {
        int a =0,b=0;

        int m = nums1.length,n= nums2.length;

        while(a<m && b<n){
            if(nums1[a]== nums2[b]){
                return nums1[a];
            }
            else if(nums1[a]<nums2[b]){
                a++;
            }
            else{
                b++;
            }
        }

        return -1;
    }
}

File: 2634-minimum-common-value/minimum-common-value.py
class Solution:
    def getCommon(self, nums1: List[int], nums2: List[int]) -> int:
        i=0
        j=0
        m=len(nums1)
        n=len(nums2)
        while(i<m and j<n):
            if(nums1[i]==nums2[j]):
                return nums1[i]
            elif (nums1[i]<nums2[j]):
                i=i+1
            else:
                j=j+1
        return -1